dandi/download.py

```diff
@@ -501,3 +502,68 @@ def _download_file(
         os.utime(path, (time.time(), ensure_datetime(mtime).timestamp()))
 
     yield {"status": "done"}
+
+
+class DownloadDirectory:
+    def __init__(self, filepath, filehash):
+        #: The path to which to save the file after downloading
```

Comment:
Is this convention `#:` some understood by any tool? just wondering

Reply:
It's the syntax for documenting attributes used by [Sphinx](https://www.sphinx-doc.org/en/master/)'s autodoc extension.